tutorials/units-and-integration/units-and-integration.ipynb

```diff
@@ -0,0 +1,234 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x7fbf1f550b38>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "nu = np.linspace(1., 3000., 1000) * u.THz\n",
+    "bb5000K_nu = blackbody_nu(in_x=nu, temperature=5000. * u.Kelvin)\n",
+    "plt.plot(nu, bb5000K_nu)\n",
+    "plt.xlabel(r'$\\nu ~ [{\\rm THz}]$')\n",
+    "plt.ylabel(r'$I_{\\nu} ~ [{\\rm erg/s/cm^2/Hz/sr}]$')\n",
+    "plt.title('Planck function in frequency')\n",
```

Comment:
For some reason I don't understand, the `~` prevents the rest of the string from being displayed in the axis labels on my machine. Removing it means there is no space between the symbol and its units...not sure I have a good solution to this...

Reply:
If it is working for other people there but not for me then it may just be some configuration thing on my end.